ydb/core/blobstorage/dsproxy/dsproxy_put.cpp

```diff
@@ -495,6 +500,19 @@ class TBlobStorageGroupPutRequest : public TBlobStorageGroupRequestActor {
                 PutImpl.Blobs[blobIdx].Recipient, PutImpl.Blobs[blobIdx].Cookie, false);
             PutImpl.Blobs[blobIdx].Replied = true;
         }
+
+        if (TActivationContext::Monotonic() - StartTime >= LongRequestThreshold) {
```

Comment:
This code handles several put requests together and will be called for every one written blob. Don't this it is a good idea to duplicate history several times in log.